dds/DCPS/Service_Participant.cpp

```diff
@@ -1664,6 +1664,8 @@ Service_Participant::load_common_configuration(ACE_Configuration_Heap& cf,
     if (got_chunk_association_multiplier) {
       ACE_DEBUG((LM_NOTICE, message, ACE_TEXT("DCPSChunkAssociationMultiplier")));
     } else {
+      // This is legacy support for a misspelling of the config option. 
+      GET_CONFIG_VALUE(cf, sect, ACE_TEXT("DCPSChunkAssociationMutltiplier"), this->association_chunk_multiplier_, size_t)
       GET_CONFIG_VALUE(cf, sect, ACE_TEXT("DCPSChunkAssociationMultiplier"), this->association_chunk_multiplier_, size_t)
     }
```

Comment:
Would Line 1669 overwrite Line 1668?

Reply:
If the user for some reason used both the old incorrect spelling and the new correct spelling, the new value would overwrite the old value. 

If the user used only the old spelling, it would not be overwritten because get_config_value would be a no-op for the correct spelling.

If the user only used the correct spelling, it would be a no-op for the old spelling and get the value from the correct spelling.